FOSSAsia/ScheduleViewController.swift

```diff
@@ -34,9 +36,10 @@ class ScheduleViewController: EventsBaseViewController, SwipeToFavoriteCellPrese
         super.viewDidLoad()
 
         viewModel?.delegate = self
-        refreshControl.attributedTitle = NSAttributedString(string: "Pull to refresh")
-        refreshControl.addTarget(self, action: #selector(ScheduleViewController.refreshData(_:)), for: .valueChanged)
-        tableView.addSubview(refreshControl)
+        self.refreshControl = UIRefreshControl()
+        refreshControl!.attributedTitle = NSAttributedString(string: "Pull to refresh")
```

Comment:
check `refreshControl` with `if let` or `guard let`, to be safe!

Reply:
okk @imjog but I was thinking to create another pr for resolving failed testcases !!

Reply:
Do  not create another PR if you resolved it here

Reply:
@SIMRAN1 Please check the resolved failing tests: #73

Reply:
okk @imjog I have seen #73 !!